imagetool/src/main/java/com/oracle/weblogic/imagetool/util/HttpUtil.java

```diff
@@ -120,25 +116,64 @@ public static HttpClient getOraClient(String userId, String password) {
         config.setCookieSpec(CookieSpecs.STANDARD);
 
         CookieStore cookieStore = new BasicCookieStore();
-        CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
-
-        if (userId != null && password != null) {
-            BasicClientCookie cc = new BasicClientCookie("oraclelicense", "a");
-            cc.setDomain("edelivery.oracle.com");
-            cookieStore.addCookie(cc);
-            credentialsProvider.setCredentials(AuthScope.ANY, new UsernamePasswordCredentials(
-                    userId, password));
+
+        String proxyHost = System.getProperty("https.proxyHost");
+        String proxyPort  = System.getProperty("https.proxyPort");
+        HttpClient result;
+        if (proxyHost != null) {
+            result = HttpClientBuilder.create()
+                .setDefaultRequestConfig(config.build())
+                .setRetryHandler(retryHandler())
+                .setProxy(new HttpHost(proxyHost, Integer.parseInt(proxyPort)))
+                .setUserAgent("Wget/1.10")
+                .setDefaultCookieStore(cookieStore).useSystemProperties()
+                .build();
+
+        } else {
+            result = HttpClientBuilder.create()
+                .setDefaultRequestConfig(config.build())
+                .setRetryHandler(retryHandler())
+                .setUserAgent("Wget/1.10")
```

Comment:
I see that this isn't a change in this PR, but it seems a little weird to copy a user agent value from another tool. What are we using "Wget/1.10"?

Reply:
It's the orion api new security requirement.